packages/light-client/src/utils/bls.ts

```diff
@@ -0,0 +1,10 @@
+import bls, {init} from "@chainsafe/bls/switchable";
+import {IBls, Implementation} from "@chainsafe/bls/types";
+
+export function getBls(): IBls {
+  return bls;
+}
+
+export async function initBls(impl: Implementation = "herumi"): Promise<void> {
```

Comment:
Why is this required now, and was not needed with the SWIG-based blst-native implementation?

Reply:
The browser tests stopped passing. Had to add that in. It may be due to tree shaking with how I implemented the new async functions. This can be refactored in a separate pr after I fix bls and republish. Does not effect lodestar. Only tree shaking in light client.

Reply:
The browser tests stopped passing. Had to add that in. It may be due to tree shaking with how I implemented the new async functions. This can be refactored in a separate pr after I fix bls and republish. Does not effect lodestar. Only tree shaking in light client.

Reply:
Alternative will be we can wait till next release which is fine if we choose to go that direction. I will suss this out further either way though.  I think it will need to be there but could be wrong and the part that it wasn't there before tells me it's possible I'm wrong

Reply:
I think this is the line causing the issue with the tree shaking. 
 https://github.com/ChainSafe/bls/blob/a5aa94c617b2abc26ff43c2f0ca7161eabe29cd6/src/functional.ts#L1

I don't actually think the herumi selection is necessary but was part of the debug process. It "made tests work" but can likely be removed